lib/v2/ieee-security/sp.js

```diff
@@ -0,0 +1,53 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+const url = 'https://www.ieee-security.org/';
+// https://www.ieee-security.org/TC/SP2023/program-papers.html
+
+module.exports = async (ctx) => {
+    let items = [];
+    let promises = [];
+
+    const last = new Date().getFullYear() - 2000 + 1;
+
+    for (let index = 20; index < last; index++) {
+        promises.push(
+            got
+                .get(`${url}/TC/SP20${index}/program-papers.html`)
+                .then((response) => response.body)
+                .catch((e) => {
+                    // ignore 404
+                    /* deepscan-disable */
+                    // eslint-disable-next-line no-use-before-define
+                    e;
+                })
+        );
+    }
```

Comment:
```suggestion
    const last = new Date().getFullYear() + 2;
    const urlList = Array.from({ length: last }, (_, v) => `${url}TC/SP${v + 2020}/program-papers.html`);
    const responses = await got.all(urlList.map((url) => got(url)));
```
- Since you've mentioned articles will fetch from 2020, I think you may use `2020` instead of `2000`
- `url` defined contains a trailing slash so we can strip the leading slashing in `urlList`